### PR TITLE
Telemetry attribute update

### PR DESCRIPTION
**Pull request by ousmaneo**
These changes add `app_pathname`,  `app_theme` to the telemetry props. We ensure that event capture is done only when the property value is set. Pathname for the Dashboard page and Stream is also normalized to remove the Id.

Fix Graylog2/graylog-plugin-enterprise#5346 Graylog2/graylog-plugin-enterprise#5348 Graylog2/graylog-plugin-enterprise#5349 Graylog2/graylog-plugin-enterprise#5347
/nocl

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/rules/RuleSimulation.tsx [hunk 1]
```diff
@@ -117,40 +117,40 @@ const RuleSimulation = ({ rule: currentRule }: Props) => {
...
-            <MessageShow message={ruleSimulationResult} />
-          </MessageShowContainer>
-          )}
-        </>
+        )}
+        {startRuleSimulation && (
+          <>
+            <Input id="message"
+                   type="textarea"
+              // eslint-disable-next-line quotes
```
Comment (grotlue) [inline graylog2-web-interface/src/components/rules/RuleSimulation.tsx:130]: The indent of this is off.

**Review by grotlue — CHANGES_REQUESTED** [review]
Code looks good!

There was one issue while testing.

✅ `app_theme` property is sent in `groupidentify` and other events (https://github.com/Graylog2/graylog-plugin-enterprise/issues/5347)
Note: app_theme value is `noir` and `teint` instead of `light` and `dark`, which I think is fine as we call the variants like that.

✅ `app_pathname` is sent in `groupidentify` events (https://github.com/Graylog2/graylog-plugin-enterprise/issues/5349)

✅ `app_pathname` property does not contain ids anymore (https://github.com/Graylog2/graylog-plugin-enterprise/issues/5346)

❌ `pageview` event is only sent once (https://github.com/Graylog2/graylog-plugin-enterprise/issues/5348)
This worked for most cases, but when changing the theme it is sending a `pageview` event twice (both with the new theme)

**Comment by ousmaneo** [comment]
Hi @grotlue thank you for reviewing.

> ❌ pageview event is only sent once (https://github.com/Graylog2/graylog-plugin-enterprise/issues/5348)
This worked for most cases, but when changing the theme it is sending a pageview event twice (both with the new theme)

This is not related to the pageview telemetry, some page/actions trigger page one or more page renders like changing the theme.

**Comment by grotlue** [comment]
> Hi @grotlue thank you for reviewing.
> 
> > ❌ pageview event is only sent once ([Graylog2/graylog-plugin-enterprise#5348](https://github.com/Graylog2/graylog-plugin-enterprise/issues/5348))
> > This worked for most cases, but when changing the theme it is sending a pageview event twice (both with the new theme)
> 
> This is not related to the pageview telemetry, some page/actions trigger page one or more page renders like changing the theme.

Alright, in this case, let's merge it!

**Review by grotlue — APPROVED** [review]

**Pull request merged**